### PR TITLE
Improve multipath identifiers

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

**Comment by rvykydal** [comment]
Validation tests are failing because of dependencies:
```
FAILED unit_tests/pyanaconda_tests/test_kickstart_specification.py::ModuleSpecificationsTestCase::test_all_commands
FAILED unit_tests/pyanaconda_tests/test_kickstart_specification.py::ModuleSpecificationsTestCase::test_useless_command_map
FAILED unit_tests/pyanaconda_tests/modules/payloads/payload/test_rpm_ostree.py::RPMOSTreeKickstartTestCase::test_ostree_kickstart
FAILED unit_tests/pyanaconda_tests/modules/payloads/payload/test_rpm_ostree.py::RPMOSTreeKickstartTestCase::test_priority_kickstart
```
waiting for this anaconda update: https://github.com/rhinstaller/anaconda/pull/5399
```
FAILED unit_tests/pyanaconda_tests/modules/storage
/test_module_device_tree.py::DeviceTreeInterfaceTestCase::test_get_zfcp_device_data
```
waiting for python-blivet 3.6.0-13 appearing in nighthly composes.

**Comment by rvykydal** [comment]
So now only the tests depending on https://github.com/rhinstaller/anaconda/pull/5399 are failing, as expected. 
```
=========================== short test summary info ============================
FAILED unit_tests/pyanaconda_tests/test_kickstart_specification.py::ModuleSpecificationsTestCase::test_all_commands
FAILED unit_tests/pyanaconda_tests/test_kickstart_specification.py::ModuleSpecificationsTestCase::test_useless_command_map
FAILED unit_tests/pyanaconda_tests/modules/payloads/payload/test_rpm_ostree.py::RPMOSTreeKickstartTestCase::test_ostree_kickstart
FAILED unit_tests/pyanaconda_tests/modules/payloads/payload/test_rpm_ostree.py::RPMOSTreeKickstartTestCase::test_priority_kickstart
====== 4 failed, 1730 passed, 3 skipped, 26 warnings in 105.35s (0:01:45) ======
```

File: pyanaconda/modules/storage/zfcp/zfcp.py [hunk 1]
```diff
@@ -84,3 +84,15 @@ def process_kickstart(self, data):
...
         """Setup the kickstart data."""
         data.zfcp.zfcp = self._zfcp_data
+        # So far, the data contains explicit zfcp statements from a
+        # kickstart file used as input for the installation. Now, add any
+        # missing entries that come from user interaction with the GUI.
+        for fcpdev in zfcp.fcpdevs:
+            zd = data.zfcp.dataClass()
+            zd.devnum = fcpdev.devnum
+            if "wwpn" in dir(fcpdev):
+                zd.wwpn = fcpdev.wwpn
```
Comment (poncovka) [inline pyanaconda/modules/storage/zfcp/zfcp.py:94]: This code is ported from upstream, isn't it? It should be ideally `zd.wwpn = getattr(fcpdev, "wwpn", "")` and same for `fcplun`, but it doesn't make sense to change the original commits here.

**Review by poncovka — APPROVED** [review]
LGTM. Thanks!

**Review by jkonecny12 — APPROVED** [review]

File: pyanaconda/modules/storage/zfcp/zfcp.py (hunk 1 above)
Comment (rvykydal) [reply pyanaconda/modules/storage/zfcp/zfcp.py:94]: I think we can change it here, I'll open also a PR for upstream.

**Pull request merged**